decidim-meetings/app/packs/stylesheets/decidim/meetings/_item.scss

```diff
@@ -175,3 +175,107 @@
     }
   }
 }
+
+// layout reset stuff
+.meeting-poll__layout {
```

Comment:
I don't like this approach but I understand that as the mobile and desktop layouts are too different, we don't have a better way of doing this at the moment.

Reply:
Yes, the mobile version uses a very different layout, maybe a future approach is to define a clean layout, but it would be used only in this case so that would be kinda overkill